cijoe/scripts/benchmark_normalize.py

```diff
@@ -65,19 +71,43 @@ def extract_bdevperf(args, cijoe, step):
     if not search:
         return errno.EINVAL
 
+    artifacts = args.output / "artifacts"
+
     collection = []
 
     search = Path(search)
-    for path in search.rglob(f"{BDEVPERF_OUTPUT_PREFIX}*.txt"):
-        with path.open() as ofile:
-            match = re.match(OUTPUT_REGEX, str(path))
+    for bdevperf_path in search.rglob(f"{BDEVPERF_OUTPUT_PREFIX}*.txt"):
+        linuxperf_path = bdevperf_path.with_name(
+            bdevperf_path.name.replace(BDEVPERF_OUTPUT_PREFIX, LINUXPERF_OUTPUT_PREFIX)
+        )
+
+        cpu = 0
+
+        with (linuxperf_path).open() as ofile:
+            file_content = ofile.read()
+            matches = re.findall(LINUXPERF_LINE_REGEX, file_content)
+            if len(matches) != 2:
+                log.error(
+                    f"Unexpected amount of CPU usage percentages (expected 2) in {linuxperf_path.name}: {matches}"
+                )
+                if len(matches) < 2:
+                    log.error("Too few arguments, needs at least 2.")
```

Comment:
Maybe this error message is a little misleading. This seems to indicate that someone could have done something different, but it would be an error in perf right?